src/styles/_custom-theme.scss

```diff
@@ -74,7 +81,7 @@ $theme-color-secondary-darker: $custom-secondary-darker;
 
 // currently, accent cool is the same as primary
 //if you have the need for accent color palettes, you can set them here
-$theme-color-accent-cool-lighter: white; //set to white in order to access white in mixins
```

Comment:
Turns out I can access white in mixins without setting it as one of the theme colors, I guess it is its own color token